### PR TITLE
Dependency updates and other minor changes

### LINKED ISSUE #923: Update testcontainers version to `1.19.0`
https://github.com/testcontainers/testcontainers-java/releases/tag/1.19.0

### PR DESCRIPTION
**Pull request by mialbu**
Closes #923

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: neow3j-integration
 
 on:
   pull_request:
-    types: [ assigned, review_requested, ready_for_review ]
+    types: [ review_requested, ready_for_review ]
```
Comment (gsmachado) [inline .github/workflows/integration.yml:5]: Why did you remove this?

It's an easy way to re-trigger the integration test if there's any error. I would keep it. I remember when @csmuller and I decided to do this, because there was no easy way to retrigger only the int tests...

**Comment by mialbu** [comment]
@gsmachado fyi, I removed the `assigned` trigger for the integration test workflow. Whenever a PR is opened, a review is requested and an assignee is set. Then, this workflow is triggered three times, which does not make sense.

File: .github/workflows/integration.yml (hunk 1 above)
Comment (mialbu) [reply .github/workflows/integration.yml:5]: You can just as easily convert it to `draft` and mark it as `ready for review` again.

Comment (mialbu) [reply .github/workflows/integration.yml:5]: And hence not trigger it three times whenever you open a new PR.

**Review by gsmachado — APPROVED** [review]
This PR doesn't pose any major risks to the neow3j core functionality. So, it's fine to merge! 🎉
